### ISSUE TITLE
Add new accessibilityFeature metadata field to FileSets

### ISSUE THREAD

**Issue by CGillen** [OregonDigital/OD2#3454]
### Descriptive summary
As part of the accessibility remediation work, the accessibility would like a new metadata field using the `schema:accessibilityFeature` predicate to be added to FileSet metadata. This field is to signal if the fileset is an accessible version of another existing FileSet. The existing FileSet will be left in place and the new metadata field does not need to record which FileSet it remediated. The field should be a boolean value, T/F.

### Expected behavior
A new boolean field is added to FileSet metadata, using the `schema:accessibilityFeature` predicate.
The field is editable on the FileSet edit page.
As of yet, the field does not need to be visible outside of the edit form.
